.github/workflows/e2e.yml

```diff
@@ -0,0 +1,48 @@
+# e2e test for Wp-menu-custom-fields
+
+name: CI for Wp-menu-custom-fields
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [automation/playwright]
```

Comment:
@alvitazwar Update the branch name to develop. As we are merging this PR in develop and any PR raised against develop we are going to run tests on that branch.